### PR TITLE
Feat : Trash Notes With API Integration

### PR DESCRIPTION
**Pull request by Rishi05051997**
Peer members, please review Notes Trash Functionality which contains:

- Trash Notes layout
- Trash Notes API Integration
trash-notes

### CODE REVIEW

**Review by amirnathani9 — APPROVED** [review]
No changes are needed. Good to Merge.
Things I Like are-

- Clean and Understandable code.
- Proper Spacing.
- File structure is good

File: src/components/Trash/TrashCard.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import ReactQuill from "react-quill";
+import { useNavigate } from "react-router-dom"
+import { Icon } from "@iconify/react"
+import { useData } from "../../context/DataProvider";
+import { deleteNoteFromTrashById, MoveFromTrashToNote } from "../../services";
+// import { deleteLableHandler } from "../../services";
+// import Select from 'react-select';
```
Comment (SanketDhabarde) [inline src/components/Trash/TrashCard.jsx]: Remove commented code

File: src/components/Trash/TrashCard.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                        {
+                            note.selectedLabels.map(({ value }, i) =>
+                                <button key={i} className="chips-btn" >
+                                    <p>{value}</p> <Icon className="iconify cursor_" onClick={(e) => deleteLableHandler(e, value, setLabelArr)} icon="akar-icons:circle-x" />
+                                </button>
+                            )
+                        }
+
+                    </div>
+                } */}
```
Comment (SanketDhabarde) [inline src/components/Trash/TrashCard.jsx]: Remove this as well

File: src/components/Trash/TrashCard.jsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+                                icon="fluent:pin-off-16-filled"
+                            />
+
+                    }
+                    {/* <Select
+                        closeMenuOnSelect={false}
+                        isMulti
+                        options={selectedLabels}
+                        onChange={(e) => ({ ...note, selectedLabels: [...note.selectedLabels].concat(e) })}
+                    /> */}
```
Comment (SanketDhabarde) [inline src/components/Trash/TrashCard.jsx]: Remove

File: src/services/trash-service.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+            dispatch({ type: "DELETE-FROM-TRASH", payload: _id })
+            return archives;
+        }
+    } catch (error) {
+        dispatch({ type: "SHOW_TOAST", payload: "Something went Wrong in deleting note from Trash" });
+    }
+}
+
+export const MoveFromTrashToNote = async (dispatch, _id, noteData, token, navigate) => {
+    debugger;
```
Comment (SanketDhabarde) [inline src/services/trash-service.js]: Remove the debuggers as well.

**Review by SanketDhabarde — APPROVED** [review]
- Code looks clean
- Like that, you have created separate files for the services
- properly indented

File: src/components/Trash/TrashCard.jsx (hunk 1 above)
Comment (Rishi05051997) [reply src/components/Trash/TrashCard.jsx]: Done

File: src/components/Trash/TrashCard.jsx (hunk 3 above)
Comment (Rishi05051997) [reply src/components/Trash/TrashCard.jsx]: Done

File: src/services/trash-service.js (hunk 4 above)
Comment (Rishi05051997) [reply src/services/trash-service.js]: Done

**Pull request merged**